addons/l10n_au/data/account_tax_report_data.xml

```diff
@@ -78,6 +79,7 @@
 
     <record id="account_tax_report_gstrpt_g9" model="account.tax.report.line">
         <field name="name">G9: GST on sales (G8 divided by eleven)</field>
+        <field name="code">G9</field>
```

Comment:
This is only supported since version 13.0

https://github.com/odoo/enterprise/commit/5d7ed69e423d6e6814a5a185279afd3b634c531b

Reply:
Oh dear, sorry - but when it was green on all tests?  And this is v13?

OK, will check it again

Reply:
> richard-willdooit wants to merge 1 commit into odoo:13.0 from WilldooIT:13.0-l10n_au-tax-headings

Yes :)

Regarding the tests, the localization were very poorly tested until version 14.0, and really tested only since version 15.0. Sorry about that, but everything merged before version 15.0 needs to be tested manually.

Reply:
Well, I hope this one nails it now.  I created a v13 db and tested it.